### ISSUE TITLE
transfer_type = 1

### ISSUE THREAD

**Issue by skinkie** [google/transit#500]
### Ask a question

For transfer_type = 1 the documentation states: _Timed transfer point between two routes. The departing vehicle is expected to wait for the arriving one and leave sufficient time for a rider to transfer between routes._

That having said; shouldn't the from_route_id and to_route_id not being conditionally required (instead of optional)? Something like: optional if from_trip_id (or to_trip_id) is specified.

I am missing the situation where *_route_id would be _required_.

**Comment by gcamp** [comment]
The way I understand it, if `*_route_id` is not present it just means all the route servicing the two `*_stop_id` are waiting for each other. That's potentially just one per stop, but the transfer doesn't need to specify it.

**Comment by skinkie** [comment]
The question is specifically: _1 - Timed transfer point between two **routes**._

**Comment by gcamp** [comment]
Yes that's what I'm talking about. Ff you transfer from stop A (serving route 1) to stop B (serving route 2), is that a transfer between two routes? 

"transfer between two routes" is used interchangeably with "transfer" in the spec (same for type `0` and `3`).

**Comment by skinkie** [comment]
So adding a route_id makes it more _specific_, hence between a _specific_ feeder and distributor? And omitting it, means any routes?
